packages/react-components/src/components/styles/ReactionButton.styles.ts

```diff
@@ -82,3 +82,49 @@ export const reactionToolTipHostStyle = (): ITooltipHostStyles => {
     }
   };
 };
+
+/* @conditional-compile-remove(reaction) */
+/**
+ *
+ * @private
+ */
+export const mobileViewMenuItemStyle = (): React.CSSProperties => {
+  return {
+    display: 'flex',
+    justifyContent: 'space-between',
```

Comment:
on landscape - does this mean they are all really far apart?

Reply:
yes. not really really far apart, but thats the design, I guess.